apps/yapms/src/routes/app/+layout.svelte

```diff
@@ -14,7 +14,7 @@
 		const map = encodeURIComponent($page.url.searchParams.get("m") ?? "");
 		// if the user is not loading a map, redirect them to the default map
 		if (map === "") {
-			await goto(`/app/usa/presidential/2022`);
+			await goto($page.url);
 			return;
 		}
```

Comment:
I think we need this so the /app url gets redirected to something if the m= is missing. But actual pages don't get redirected.
```ts
if (map === "") {

	if ($page.url.pathname === "/app") {
		await goto("/app/usa/presidential/2022");
	}

	return;
}
```